protobuf/lib/src/protobuf/mixins/well_known.dart

```diff
@@ -1,8 +1,11 @@
+import 'dart:convert';
+
```

Comment:
You could consider linking to where it is documented how well-known types should be proto3 JSON encoded.